scripts/simex

```diff
@@ -67,8 +67,54 @@ def cli_selects_run(args, run):
 	if args.instset is not None:
 		if args.instset not in run.instance.instsets:
 			return False
+	if args.repetition is not None:
+		if args.repetition != run.repetition:
+			return False
 	if args.run is not None:
-		if args.run != run.experiment.name + '/' + run.instance.shortname:
+
+		m = re.search(r'(?P<experiment>[^~@/\[\]]+)'
+						r'(~(?P<variants>[^~@/\[\]]+))?'
+						r'(@(?P<revision>[^~@/\[\]]+))?'
+						r'(/(?P<instance>[^~@/\[\]]+))'
+						r'(\[(?P<repetition>\d+)\])?', args.run)
```

Comment:
We should compile this regexp outside of the cli_selects_run function to avoid redundant work.

Reply:
Done.